.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
 - "1.12"
+go_import_path: github.com/superbrothers/kubectl-open-svc-plugin
```

Comment:
Why do you add this line? Do you have any problem?

Reply:
I faced the build failed that could not find internal package imports in travis-ci. So that I added it in conf following document. Then It certainly passed travis tests.
cf: https://docs.travis-ci.com/user/languages/go/#go-import-path
```
if vanity imports are necessary (especially for internal package imports), go_import_path: may be specified at the top level of the config
```